modules/ssh/ssh.go

```diff
@@ -317,7 +319,11 @@ func Listen(host string, port int, ciphers, keyExchanges, macs []string) {
 		}
 	}
 
-	go listen(&srv)
+	go func() {
+		_, _, finished := process.GetManager().AddTypedContext(graceful.GetManager().HammerContext(), "Service: SSH", process.SystemProcessType, true)
```

Comment:
"SSH" feels to generalized. Could be confused with a incoming SSH connection or something alike.
```suggestion
		_, _, finished := process.GetManager().AddTypedContext(graceful.GetManager().HammerContext(), "Service: Built-in SSH server", process.SystemProcessType, true)
```